src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -273,6 +274,11 @@
         <xsd:attribute name="enabled" type="xsd:boolean" />
     </xsd:complexType>
 
+    <xsd:complexType name="php-errors">
+        <xsd:attribute name="log" type="xsd:boolean" />
```

Comment:
given this should allow bool|int on 4.1, how should the xsd look like there? (to ease merging up to master)

Reply:
see #26740